ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineNewPayload.java

```diff
@@ -575,19 +575,31 @@ protected ValidationResult<RpcErrorType> validateBlobs(
 
     // Validate excessBlobGas
     if (maybeParentHeader.isPresent()) {
-      if (!validateExcessBlobGas(header, maybeParentHeader.get(), protocolSpec)) {
+      Optional<BlobGas> maybeCalculatedExcess =
+          validateExcessBlobGas(header, maybeParentHeader.get(), protocolSpec);
+      if (maybeCalculatedExcess.isPresent()) {
```

Comment:
this extra if may change the logic?

Reply:
the if statement is doing the same validation as before. The only change I had made was to have the two validation methods return Optionals instead of a boolean. An `Optional.empty()` would be the same as a boolean true from the previous logic, and instead of return a false like before, it would return an Optional having the calculated value required. I will add a comment explaining this to the methods I have modified. Please let me know if this change is okay.